templates/repo/actions/runs_list.tmpl

```diff
@@ -36,15 +36,20 @@
 					<div class="run-list-meta">{{svg "octicon-calendar" 16}}{{DateUtils.TimeSince $run.Updated}}</div>
 					<div class="run-list-meta">{{svg "octicon-stopwatch" 16}}{{$run.Duration}}</div>
 				</div>
-				{{if and ($.AllowDeleteWorkflowRuns) ($run.Status.IsDone)}}
-					<button class="btn interact-bg link-action tw-p-2"
-						data-url="{{$run.Link}}/delete"
-						data-modal-confirm="{{ctx.Locale.Tr "actions.runs.delete.description"}}"
-						data-tooltip-content="{{ctx.Locale.Tr "actions.runs.delete"}}"
-					>
-						{{svg "octicon-trash"}}
-					</button>
-				{{end}}
+				<div class="ui jump dropdown item tw-p-2">
```

Comment:
Why use `item` class here?

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/34530/commits/81a275a727ee4cfdb0c8012f6e81b30fb095fa23